app/pods/contests/college/all/archived/controller.js

```diff
@@ -0,0 +1,42 @@
+import Controller from '@ember/controller';
+import { action, computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { restartableTask } from 'ember-concurrency-decorators';
+
+export default class ArchivedController extends Controller {
+  @service store
+  @service router
+  @service api
+
+  queryParams = ['offset', 'limit']
+  offset = 0
+  limit = 6
+
+  @restartableTask fetchContestsTask = function* () {
+    try {
+      return yield this.store.query('college_contest', {
+        page: this.page,
+        custom: {
+          ext: 'url',
+          url: 'archived',
+
+        }
+      })
+    } catch (err) {
+      console.log(err)
+      this.set('showError', true)
+    }
+  }
+  @action
+  setOffset(offset) {
+    this.set('page.offset', offset)
+    this.fetchContestsTask.perform()
+  }
+  @computed('offset', 'limit')
```

Comment:
we usually follow(in our projects) declaring computed properties at the top after services in chronological order. so shift it there